sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/CopyAuthorization.java

```diff
@@ -87,12 +94,12 @@ public String getAccessToken() {
     }
 
     /**
-     * Get the time when the access token expires. The date is represented as the
+     * Get the date-time when the access token expires. The date is represented as the
```

Comment:
maybe remove the last part for the comment as it is now a date and not the seconds?